sql/sql_parse.cc

```diff
@@ -7162,7 +7194,7 @@ bool add_field_to_list(THD *thd, LEX_STRING *field_name, enum_field_types type,
                        LEX_STRING *comment,
 		       char *change,
                        List<String> *interval_list, const CHARSET_INFO *cs,
-		       uint uint_geom_type)
+		       uint uint_geom_type, const LEX_STRING *zip_dict)
```

Comment:
changed `LEX_STRING` -> `LEX_CSTRING` in all zip_dict-related function signatures.